src/Compilers/CSharp/Test/Semantic/Semantics/FunctionPointerTests.cs

```diff
@@ -29,6 +29,13 @@ private CompilationVerifier CompileAndVerifyFunctionPointers(CSharpCompilation c
             return CompileAndVerify(compilation, verify: Verification.Skipped, expectedOutput: expectedOutput);
         }
 
+        [Fact]
+        public void UsingTest()
+        {
+            CreateCompilationWithFunctionPointers(@"
+using s = delegate*<void>;").VerifyDiagnostics();
```

Comment:
I would expect us to try using alias in a meaningful way, compile and run the code. Could be followed up on in a separate PR.

Reply:
It looks like this test is failing in CI, a change is needed anyway.

---
In reply to: [432721087](https://github.com/dotnet/roslyn/pull/44564#discussion_r432721087) [](ancestors = 432721087)